assets/extensions/update-notification/woocommerce-notice.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _n, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Multiple from './multiple';
+import UpdateAvailableLabel from './update-available-label';
+import { Col } from '../grid';
+import { UpdateIcon } from '../../icons';
+import ExtensionActions from '../extension-actions';
+
+/**
+ * WooCommerce notice component.
+ *
+ * @param {Object} props             Component props.
+ * @param {Array}  props.extensions  Extensions list.
+ * @param {Array}  props.isConnected Whether the site is connected to WC.com.
+ */
+const WooCommerceNotice = ( { extensions, isConnected } ) => {
```

Comment:
Just referencing a related comment in the other PR: https://github.com/Automattic/sensei/pull/4213#discussion_r626882566

Reply:
Was that the wrong link? I think you meant this one - https://github.com/Automattic/sensei/pull/4213#discussion_r626889790? If so, I think it's safe to skip doing that now.

Reply:
Ah, yep! Not a big deal! :)